suite/suite.go

```diff
@@ -16,21 +16,29 @@ import (
 var allTestsFilter = func(_, _ string) (bool, error) { return true, nil }
 var matchMethod = flag.String("testify.m", "", "regular expression to select tests of the testify suite to run")
 
+type TestingT interface {
+	Run(name string, f func(t *testing.T)) bool
+	Helper()
```

Comment:
I wouldn't limit the interface unnecessarily. If I'm not mistaken, `Helper()` isn't used anywhere in `Suite`; would it not make sense to leave it out for now until absolutely necessary?